workers/constructors/SharedWorker/same-origin.html

```diff
@@ -19,7 +19,7 @@
       assert_true(e instanceof Event);
     });
   } catch (e) {
-    t.step_func_done(function(e) { assert_true(true); });
+    t.done();
```

Comment:
You want the assert_throws here too, and in the third test.

Reply:
Sigh...of course, whoops